app/templates/components/hyper-table/cell-renderers/image.hbs

```diff
@@ -5,7 +5,9 @@
 {{#if (gt column.labels.length 0)}}
   <div>
     {{#each column.labels as |label|}}
-      <span>{{get item label}}</span>
+      <span data-toggle="tooltip" data-title={{get item label}}>
```

Comment:
there is a real gain to add the same value as `data-title` ?

Reply:
it's because the column can be truncated with ellipsis

Reply:
good point :+1: